scripts/lib/clean_unused_caches.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
Removing the shebang line is incorrect; it is still possible to run it independently.

Also, I think you forgot to push after the above feedback.

Reply:
Sorry for another round of follow-up.

> You need to update things that call this script when renaming it.

I changed the parse script args description to  'This ~script~ file cleans unused Zulip caches.'
Do I also need to change this file location to avoid having `scripts/` in its location name? 
And anything else than this?

> Removing the shebang line is incorrect; it is still possible to run it independently.

I added it back. Can you please explain to me a bit more what needs to be done, I am quite confused?
We can currently run the file via `python3 scripts/lib/clean_unused_caches.py` or `./scripts/lib/clean_unused_caches.py`.